woocommerce/woocommerce-gutenberg-products-block
# Import an sql to prepare e2e tests.

This PR adds the ability to import a database to our e2e test, I've already created one with those things:

- Different taxes for different countries with different values. 
- Enabled shipping only for certain countries.
- Added some shipping methods to a couple of zones (two methods for US + a free shipping method that is activated by a coupon, a shipping method for UK and a global shipping method with free shipping if the cart is above $400)
- Several coupons with several configurations.
- Made a product sold by individually.

It also removes some prep tests that imported products and deleted other products, since we reset the DB this should be accounted for in e2e.
### How to test
1. destroy your docker by running `npm run docker:down:reset`
2. up it again `npm run docker:up:build`
3. visit http://localhost:8084/wp-admin `admin:password`
4. confirm all previous things.
4. run `npm run test:e2e` to make sure nothing is broken.

### How to generate a DB dump

1. get the MySQL container ID `docker container ls`
2. use it instead of CONTAINER_ID
```
docker exec CONTAINER_ID /usr/bin/mysqldump -u wordpress --password=wordpress wordpress > tests/bin/db.sql
```
closes #2053
